docs/README.md

```diff
@@ -72,43 +68,65 @@ Functionally, the app has several key features:
 
 [More about how/why to interact with web3 here](https://gitcoin.co/web3).
 
-Technically, the system is architected:
+## Technically
```

Comment:
Could we make it `###` as it's a subcategory of `What is Gitcoin` ?a

Reply:
done